src/main/java/tr/com/orioninc/laborant/service/LabService.java

```diff
@@ -44,32 +49,34 @@ public static String connectAndExecuteCommand(String username, String password,
         }
         catch(Exception e)
         {
+            log.error("[connectAndExecuteCommand] Error occured while connecting to lab: {}" , e.getMessage());
             e.printStackTrace();
         }
         finally {
             if (session != null) {
                 session.disconnect();
+                log.info("[connectAndExecuteCommand] Session disconnected");
             }
             if (channel != null) {
+                log.info("[connectAndExecuteCommand] Channel disconnected");
                 channel.disconnect();
             }
-            return responseString;
         }
-
+        log.info("[connectAndExecuteCommand] Response string: {}", responseString);
+        return responseString; // it was inside finally block
     }
 
     public String getALlLabsStatus(){
```

Comment:
Function name is not following the naming convention
```suggestion
    public String getAllLabsStatus(){
```

Reply:
refactored